src/pages/docs/creators/ray/quickstart.md

```diff
@@ -0,0 +1,100 @@
+---
+description: quick and easy, copy and run instructions on launching and decommissioning the Ray on Golem cluster
+title: Ray on Golem QuickStart
+type: guide 
+---
+
+# Quickstart
+
+This quickstart shows you how to set Ray and Ray on Golem up, start your cluster, test it, and then stop it.
+It limits the explanation to the bare minimum - if you are looking for more details jump to [setup tutorial](/docs/creators/ray/setup-tutorial)
+
+
+## Install software
+
+The first step is installing Ray and Ray on Golem (recommended within a clean virtual environment)
+
+```bash
+# install ray & ray-on-golem
+pip install -U ray[default] ray-on-golem
+```
+
+For now, you also need to download and install Golem node software representing you in the Golem network.
+
+```bash
+# install yagna - golem network daemon
+curl -sSf https://join.golem.network/as-requestor | bash -
+```
+
+## Start `ray-on-golem` server
+
+For the time being you need to manually run `ray-on-golem` server (in a separate terminal)
+
+```bash
+ray-on-golem
+```
+
+## Set the cluster up
+
+With the packages in place, you can download our sample golem cluster configuration yaml, and use it with `ray up` to start up the cluster.
+It will give you a cluster of one node (which will expand when you feed it with work) on the Golem test network (free, but not very powerful)
+
+
+```bash
+# Download the golem-cluster.yaml
+wget https://github.com/golemfactory/golem-ray/blob/main/golem-cluster.yaml
```

Comment:
outdated link

https://github.com/golemfactory/ray-on-golem/blob/main/golem-cluster.yaml

is works

I am not sure we want to point to main

Reply:
fixed - we use https://github.com/golemfactory/ray-on-golem/raw/main/golem-cluster.yaml
where would you point?
I thought it will be a natural place for the most up to date example yaml that will start the cluster all right